scripts/core/common.yml

```diff
@@ -102,7 +102,7 @@ name: "$x_kernel_handle_t"
 --- #--------------------------------------------------------------------------
 type: handle
 desc: "Handle of a queue object"
-class: $xPlatform
+class: $xQueue
```

Comment:
this needs to be moved into another PR. Otherwise this one lgtm and we can merge it.

Reply:
Done. The queue fix is here https://github.com/oneapi-src/unified-runtime/pull/1285.